scala-debugger-api/src/main/scala/org/scaladebugger/api/profiles/scala210/info/Scala210FieldTransformationRules.scala

```diff
@@ -0,0 +1,42 @@
+package org.scaladebugger.api.profiles.scala210.info
+
+import org.scaladebugger.api.profiles.traits.info.FieldVariableInfo
+
+trait Scala210FieldTransformationRules {
+  // Will retrieve fields of these objects
+  private val expandNames = Seq("$outer")
+
+  // Will skip these fields
+  // TODO: Provide better means of skipping executionStart
+  //       since this avoids it even if someone added it directly
+  private val ignoreNames = Seq("MODULE$", "executionStart", "serialVersionUID")
+
+  // Will skip fields whose names start with these prefixes
+  private val ignoreNamePrefix = Seq("scala$")
+
+  // Will skip fields whose origin starts with these strings
+  private val ignoreOrigin = Seq("scala.")
+
+  // TODO: Handle infinitely-recursive fields when expanding
+  def transformField(field: FieldVariableInfo, isInStaticContext: Boolean = false): Seq[FieldVariableInfo] = {
```

Comment:
Could this be made protected or private instead? I don't want this method to appear publicly on `Scala210ObjectInfo` and other classes that inherit it.